Services/DataBaseService/DbModels/DbUser.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+
+namespace DataBaseService.DbModels
+{
+    public class DbUser
```

Comment:
How about write summary to public classes

Reply:
Also, Is it dbUser or dbUserCredential?

Reply:
Honestly, I don`t now, what I should write in summary.

Reply:
Rename DbUser to DbUserCredential